HolyshiftService/Auth/JwtService.cs

```diff
@@ -0,0 +1,76 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using HolyShift.Config;
+using HolyShift.Database.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace HolyShift.Auth;
+
+public class JwtService
+{
+    private readonly IOptions<AuthConfig> _authConfig;
+    private readonly HolyShiftDbContext _db;
+
+    public JwtService(IOptions<AuthConfig> authConfig, HolyShiftDbContext db)
+    {
+        _authConfig = authConfig;
+        _db = db;
+    }
+
+    public string GenerateJwtToken(UserDbModel user)
+    {
+        var config = _authConfig.Value;
+        var securityKey = new SymmetricSecurityKey(Convert.FromBase64String(config.KeyBase64));
+        var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
+        var claims = new[]
+        {
+            new Claim(JwtRegisteredClaimNames.Sub, user.Id.ToString()),
+            new Claim("role", user.Role.ToString())
+        };
+
+        var token = new JwtSecurityToken(
+            config.Issuer,
+            config.Audience,
+            claims,
+            expires: DateTime.UtcNow.AddMinutes(config.AccessTokenExpirationMinutes),
+            signingCredentials: credentials
+        );
+
+        return new JwtSecurityTokenHandler().WriteToken(token);
+    }
+
+    public string GenerateRefreshToken(UserDbModel user)
```

Comment:
Let's delete refresh token since we are not using it right now and add it later.

You can keep it in a separate branch so that it is still there and we can use the code later.